src/main/webapp/app/exercises/programming/assess/repo-export/programming-assessment-repo-export-dialog.component.ts

```diff
@@ -62,6 +63,7 @@ export class ProgrammingAssessmentRepoExportDialogComponent implements OnInit {
                 .pipe(
                     tap(({ body: exercise }) => {
                         this.exercise = exercise!;
+                        this.isAtLeastInstructor = exercise!.isAtLeastInstructor ? exercise!.isAtLeastInstructor : false;
```

Comment:
```suggestion
                        this.isAtLeastInstructor = !!exercise!.isAtLeastInstructor;
```
```suggestion
                        this.isAtLeastInstructor = exercise!.isAtLeastInstructor ?? false;
```
Both would be nicer in my opinion

Reply:
Suggestion has been applied, thank you 🙂